community.md

```diff
@@ -750,9 +759,14 @@ the use of DPGs
 when their use might contravene various social values.  Regardless of
 whether or not we call these licenses "open source", people working on
 DPGs in a government context might well come across DPGs that use
-these licenses.  This is a space to watch, and for now, the DPG standard does *not*
+these licenses.  This is a space to watch, perhaps especially as AI adoption grows.
```

Comment:
I may (diplomatically) tighten up the language about these "ethical source" licenses here, after the merge.  They are an interesting experiment, but I don't see them as being particularly related to open source licensing (except in a historical lineage sense, of course -- obviously, the idea to try them came from the seeing the success of open source licensing).  No action needed here; I'm just making a note mainly to remind myself.

Reply:
ok......